eox_nelp/signals/tasks.py

```diff
@@ -132,15 +134,35 @@ def _generate_progress_enrollment_data(user, course_id, user_has_passing_grade):
     return progress_enrollment_data
 
 
+def create_external_certificate(external_certificate_data, user_id=None, course_id=None):
```

Comment:
technically this is not a task, doesn't it ?

Reply:
You are right. Is a method handler that decide witch task use.